tests/__init__.py

```diff
@@ -16,3 +16,4 @@
 # You should have received a copy of the GNU General Public License
 # along with Invenio; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+"""OAuthClient unit tests."""
```

Comment:
Actually it's needed for importing ``tests/helpers.py``: https://github.com/inveniosoftware/invenio-oauthclient/blob/master/tests/test_contrib_cern.py#L32